frontend/src/components/smart/PvPArenaMatchMaking.vue

```diff
@@ -446,8 +446,8 @@ export default {
       } catch (err) {
         console.log('leave arena error: ', err.message);
 
-        this.handleErrorMessage(err.message, 'Not in arena', i18n.t('pvp.charNotInArena'));
-        this.handleErrorMessage(err.message, 'Defender duel in process', i18n.t('pvp.duelInProcess'));
+        this.handleErrorMessage(err.message, 'N', i18n.t('pvp.charNotInArena'));
+        this.handleErrorMessage(err.message, 'Q', i18n.t('pvp.duelInProcess'));
```

Comment:
We had to reduce them heavily due to contract size limitations in the past, now that the contracts are split, the issue is not so much but it might resurge in the future